docs/OVERVIEW.md

```diff
@@ -408,21 +409,21 @@ as an OME-TIFF, the writer class can also be used to customize as needed.
 
 ```python
 import numpy as np
-from bioio.writers import OmeTiffWriter
+from bioio.writers import OmeTiffWriter # with bioio-ome-tiff installed
 
 image = np.random.rand(10, 3, 1024, 2048)
-OmeTiffWriter.save(image, "file.ome.tif", dim_order="ZCYX")
+OmeTiffWriter.save(image, "file.ome.tiff", dim_order="ZCYX")
 ```
 
 See
-[OmeTiffWriter documentation](https://bioio-devs.github.io/bioio/bioio.writers.html)
+[OmeTiffWriter documentation](https://github.com/bioio-devs/bioio-ome-tiff/blob/main/README.md)
 for more details.
 
 #### Other Writers
 
 In most cases, `BioImage.save` is usually a good default but there are other image
-writers available. For more information, please refer to
-[our writers documentation](https://bioio-devs.github.io/bioio/bioio.writers.html).
+writers available. For more information, please refer to the writer registry.
+[our writers registry](https://github.com/bioio-devs/bioio/blob/main/README.md).
```

Comment:
can you include a more specific link here that automatically scrolls to the writer registry section of the readme?

Reply:
oops I got here too late, nevermind

Reply:
Ill open a ticket!